cli/command/node/builders_test.go

```diff
@@ -0,0 +1,149 @@
+package node
```

Comment:
Yeah, I though about that, but didn't want to get too much ahead and refactor as I go in other PR. I don't want to over-design if it's not needed.